kubernetes/windows/Dockerfile

```diff
@@ -94,4 +93,17 @@ ENV AZMON_COLLECT_ENV False
 ENV CI_CERT_LOCATION "C://oms.crt"
 ENV CI_KEY_LOCATION "C://oms.key"
 
+# Delete unnecessary files after installation
+RUN powershell -Command "Remove-Item -Recurse -Force 'C:\ruby31\share\doc\ruby\html\js'; \
+Remove-Item -Force 'C:\ruby31\ridk_use\ridk.ps1'; \
+Remove-Item -Force 'C:\ruby31\msys64\usr\share\texinfo\js\info.js'; \
+Remove-Item -Force 'C:\ruby31\msys64\usr\share\texinfo\js\modernizr.js'; \
+Remove-Item -Force 'C:\ruby31\msys64\usr\share\terminfo\78\xterm.js'; \
+Remove-Item -Force 'C:\ruby31\msys64\usr\lib\terminfo\78\xterm.js'; \
+Remove-Item -Force 'C:\ruby31\lib\ruby\3.1.0\rdoc\generator\template\json_index\js\navigation.js'; \

```

Comment:
This may not be required if we install ruby with /NoDoc params.

Reply:
choco does not support noDoc params: https://community.chocolatey.org/packages/ruby